.github/workflows/moodle-ci.yml

```diff
@@ -31,15 +31,15 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - php: '8.1'
-            moodle-branch: 'master'
-            database: 'mariadb'
-          - php: '8.1'
+          - php: '8.2'
```

Comment:
So the matrix tested master branch against both types of databases, you making this only test against pgsql

we want to test both dbs on 8.2 on master